datafusion/expr/src/aggregate_function.rs

```diff
@@ -63,8 +63,6 @@ pub enum AggregateFunction {
     Stddev,
     /// Standard Deviation (Population)
     StddevPop,
-    /// Covariance (Sample)
```

Comment:
The point of this PR is to remove this variant and make it a user defined aggregate